frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/common/BuilderDrawerContainer.tsx

```diff
@@ -0,0 +1,59 @@
+import { BiLeftArrowAlt } from 'react-icons/bi'
+import { Stack, Text } from '@chakra-ui/react'
+
+import IconButton from '~components/IconButton'
+
+import {
+  setToInactiveSelector,
+  useBuilderAndDesignStore,
+} from '../../useBuilderAndDesignStore'
+import { CreatePageDrawerCloseButton } from '../CreatePageDrawerCloseButton'
+
+interface BuilderDrawerContainerProps {
+  title: string
+  content: JSX.Element
```

Comment:
nit: probably can use react's `children` prop instead, then we can just inline the content instead of assigning it to the `contenq prop

Reply:
Got it, updated to use `children` prop